dropwizard-app/delivery/src/main/java/com/wfsample/delivery/DeliveryService.java

```diff
@@ -0,0 +1,107 @@
+package com.wfsample.delivery;
+
+import com.wfsample.common.DropwizardServiceConfig;
+import com.wfsample.common.dto.PackedShirtsDTO;
+import com.wfsample.common.dto.DeliveryStatusDTO;
+import com.wfsample.service.DeliveryApi;
+
+import java.util.Queue;
+import java.util.UUID;
+import java.util.concurrent.ConcurrentLinkedDeque;
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.ThreadLocalRandom;
+import java.util.concurrent.TimeUnit;
+
+import javax.ws.rs.core.Response;
+
+import io.dropwizard.Application;
+import io.dropwizard.lifecycle.setup.ScheduledExecutorServiceBuilder;
+import io.dropwizard.setup.Environment;
+
+/**
+ * Driver for styling service which manages different styles of shirts and takes orders for a shirts
+ * of a given style.
+ *
+ * @author Srujan Narkedamalli (snarkedamall@wavefront.com).
+ */
+public class DeliveryService extends Application<DropwizardServiceConfig> {
+  /*
+   * TODO: Add a gauge to monitor the size of dispatch queue.
```

Comment:
please add this to the TODO line -
Also, consider adding relevant ApplicationTags for this metric.